zeroex/ordervalidate/order_validator_test.go

```diff
@@ -136,24 +128,48 @@ func TestBatchValidateOffChainCases(t *testing.T) {
 		ethClient, err := ethclient.Dial(constants.GanacheEndpoint)
 		require.NoError(t, err)
 
-		signedOrders := []*SignedOrder{
+		signedOrders := []*zeroex.SignedOrder{
 			&testCase.SignedOrder,
 		}
 
 		orderValidator, err := NewOrderValidator(ethClient, constants.TestNetworkID, constants.TestMaxContentLength, 0)
 		require.NoError(t, err)
 
-		validationResults := orderValidator.BatchValidate(signedOrders, areNewOrders)
-		isValid := len(validationResults.Accepted) == 1
+		offchainValidOrders, rejectedOrderInfos := orderValidator.BatchOffchainValidation(signedOrders)
```

Comment:
Refactored these tests to use the `BatchOffChainValidation` method directly, since that's what it's actually supposed to be testing.